users/config.rst

```diff
@@ -402,32 +466,34 @@ blockPullOrder
 
     Available options:
 
-    standard (default):
+    standard (default)
         The blocks of a file are split into N equal continuous sequences, where N is the number of connected
         devices. Each device starts downloading it's own sequence, after which it picks other devices
         sequences at random. Provides acceptable data distribution and minimal spinning disk strain.
 
-    random:
+    random
         The blocks of a file are downloaded in a random order. Provides great data distribution, but very taxing on
         spinning disk drives.
 
-    inOrder:
+    inOrder
         The blocks of a file are downloaded sequentially, from start to finish. Spinning disk drive friendly, but provides
         no improvements to data distribution.
 
-copyRangeMethod
+:ref:`copyRangeMethod <advanced-folder-copyrangemethod>`
     Provides a choice of method for copying data between files. This can be used to optimise copies on network
     filesystems, improve speed of large copies or clone the data using copy-on-write functionality if the underlying
     filesystem supports it.
 
-    See :ref:`folder-copyRangeMethod` for details.
+:ref:`caseSensitiveFS <advanced-folder-casesensitivefs>`
+
+junctionsAsDirs
 
 Device Element
 --------------
 
 .. code-block:: xml
 
-    <device id="5SYI2FS-LW6YAXI-JJDYETS-NDBBPIO-256MWBO-XDPXWVG-24QPUM4-PDW4UQU" name="syno" compression="metadata" introducer="false" skipIntroductionRemovals="false" introducedBy="2CYF2WQ-AKZO2QZ-JAKWLYD-AGHMQUM-BGXUOIS-GYILW34-HJG3DUK-LRRYQAR">
+    <device id="3LT2GA5-CQI4XJM-WTZ264P-MLOGMHL-MCRLDNT-MZV4RD3-KA745CL-OGAERQZ" name="syno" compression="metadata" introducer="false" skipIntroductionRemovals="false" introducedBy="">
```

Comment:
I guess the `introducedBy` example here was put there on purpose, to illustrate the ID format?  I don't think consistency with (repetition of) the whole config above is important here.